### PR TITLE
Add talk list page and link to page.

### PR DESCRIPTION
**Pull request by sublimemarch**

### CODE REVIEW

File: talk-list.html [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+          Esa Sclafani
+        </h4>
+        <div class="description">
+          Baking is a skill that we often see as being esoteric and scientific in nature.  The concept of precise proportions and measurements is intimidating to most people, even those who are otherwise comfortable in the kitchen. I, as a professional baker, wish to detail how baking doesn't need to be scary, or as precise as we've been led to believe.  If possible, I would love to talk for five minutes and answer questions for five minutes, in case people have specific queries.
+        </div>
+      </div>
+      <div class="divider"></div>
+      <div class="talk">
+        <h3 class="title">
+          word? word. | Emojis & Communication
```
Comment (thejessleigh) [inline talk-list.html]: I think you need to escape this ampersand here

**Review by thejessleigh — CHANGES_REQUESTED** [review]
Other than the ampersand I think this looks good 👍

File: talk-list.html (hunk 1 above)
Comment (sublimemarch) [reply talk-list.html]: Most modern browsers handle ampersands just fine without escaping, but I'll escape it just in case.

**Comment by sublimemarch** [comment]
Alright, I've removed the speaker who had to pull out and updated the ampersand.

**Review by thejessleigh — APPROVED** [review]

**Pull request merged**